src/App.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from "react";
+import Weather from "./Weather";
+import "./App.css";
+
+class App extends Component {
```

Comment:
This could be a functional component using hooks. Might be worth checking out, they are pretty cool :)